CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🧬 The `vast get` subcommand has been added. It can be used to retrieve
+  events by their ids from the database.
```

Comment:
💣 I want you to imagine @mavam kindly telling you to use active voice in changelog entries.